visualization/app/codeCharta/state/store.service.ts

```diff
@@ -65,6 +66,10 @@ export class StoreService {
 			this.dispatch(setIsLoadingMap(true))
 		}
 
+		if (isActionOfType(action.type, IsLoadingFileActions)) {
```

Comment:
After migration of nodeContextMenu I will introduce [effects](https://ngrx.io/guide/effects) to get rid of this custom dispatch logic (which should have been a middleware in redux world and will become an effect in ngrx world)